package-system/mcpp/mcpp_2.7.2-az.3.patch

```diff
@@ -0,0 +1,897 @@
+diff --git a/config/config.guess b/config/config.guess
+index 396482d..e3ef63f 100644
+--- a/config/config.guess
++++ b/config/config.guess
```

Comment:
Is there any way to not modify config.guess?  Its license is GPL, and modifying it causes licensing concerns.
Is this maybe just an update of it?

Reply:
did we maybe pull in a new version of mcpp or something and get a bunch of junk added?

Reply:
This patch contains the same content of the original patch, + my new code in 'support.c'

Reply:
Why do changes to `config.guess` show up in the diff then, given that your changes only purportedly change `support.c`?